src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+    private static String logo =
```

Comment:
Wow! Very creative logo, chonguschonguschongus! 😁👌👍

Reply:
hi coder! do consider supporting the project if you really like it :) [buy me a haidilao](https://i.imgur.com/hIsG3Mo.jpg)